tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/service/grpc/tool/ProtoMessageTestCase.java

```diff
@@ -88,7 +88,7 @@ public void testPrimitiveTypeProtoMessage() {
         MapValue<String, Object> bMapValue = BallerinaValues.createRecordValue(".", "Test2");
         bMapValue.put("a", "John");
         bMapValue.put("b", 1.2D);
-        bMapValue.put("c", 2.5F);
+        bMapValue.put("c", 2.5D);
```

Comment:
@daneshk I have replaced the float value with double, because we can never receive a java float in ballerina. hope this is ok?